docs/examples/multiple_stands.md

```diff
@@ -64,13 +64,91 @@ hardpy-panel -dbp 5985 -wp 8001 tests
 
 ### storing reports
 
+We can use the database to store a list of reports. This will require a separate database.
+
```

Comment:
Here and below is need to remove extra empty lines.

Reply:
Extra empty lines have been removed.